src/Core/Exceptions/DuplicateAuthRequestException.cs

```diff
@@ -0,0 +1,10 @@
+﻿namespace Bit.Core.Exceptions;
+
+public class DuplicateAuthRequestException : Exception
+{
+    public DuplicateAuthRequestException()
+        : base("An authentication request with the same device already exist.")
```

Comment:
Will this error be displayed to the user? If so, do we want them to know this?

Also, probably want to change `exist` -> `exists`

Reply:
It wouldn't be displayed to the user, but I should change it. Thanks.